Jenkinsfile

```diff
@@ -1,7 +1,8 @@
 library 'magic-butler-catalogue'
 
-def projectName = "eslint-config-logdna"
-def repo = "logdna/${projectName}"
+def PROJECT_NAME = "eslint-config-logdna"
+def REPO = "logdna/${PROJECT_NAME}"
+def TRIGGER_PATTERN = ".*@logdnabot.*"
```

Comment:
do this need a global or multi-line switch?

Reply:
good point, it probably should

Reply:
It looks like it passes the entire body of the comment. so this should be ok. It doesn't look like you can pass switches anyway.

Reply:
actually yeah, doesn't matter because this isn't anchored